testgrid/cmd/configurator/prow.go

```diff
@@ -19,13 +19,13 @@ package main
 import (
 	"fmt"
 	"path"
+	"sort"
 	"strconv"
 	"strings"
 
 	"github.com/GoogleCloudPlatform/testgrid/config"
 	"github.com/GoogleCloudPlatform/testgrid/config/yamlcfg"
 	configpb "github.com/GoogleCloudPlatform/testgrid/pb/config"
-
```

Comment:
nit: I think this spacing between the GCP imports and the k8s imports is desired